vpn/src/main/java/dummy/ui/VpnControllerActivity.kt

```diff
@@ -31,7 +31,11 @@ import com.duckduckgo.app.trackerdetection.api.WebTrackersBlockedRepository
 import com.duckduckgo.app.trackerdetection.db.WebTrackerBlocked
 import com.duckduckgo.di.scopes.AppScope
 import com.duckduckgo.mobile.android.vpn.R
-import com.duckduckgo.mobile.android.vpn.model.*
+import com.duckduckgo.mobile.android.vpn.model.DataSizeFormatter
+import com.duckduckgo.mobile.android.vpn.model.TimePassed
+import com.duckduckgo.mobile.android.vpn.model.VpnTracker
+import com.duckduckgo.mobile.android.vpn.model.dateOfLastWeek
+import com.duckduckgo.mobile.android.vpn.model.dateOfPreviousMidnight
```

Comment:
Interesting breakage when using `com.duckduckgo.mobile.android.vpn.model.*` which we had before: Anvil gets confused which `VpnPreferences` class to use... there is one we explicitly import and another that is brought in because of that wildcard.

Reply:
I think we have a rule to disallow star imports, not sure if it is enforced.

Reply:
I think * imports should be avoided. Not sure if we've got a rule but I'd love to have it an enforce it 😬

Reply:
I'll leave y'all to have that discussion in the new year and look forward to coming back to a lovely formatted codebase 😎